op-node/rollup/superchain.go

```diff
@@ -4,12 +4,16 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/ethereum/go-ethereum/params"
+
 	"github.com/ethereum/go-ethereum/common"
 
 	"github.com/ethereum-optimism/optimism/op-service/eth"
 	"github.com/ethereum-optimism/superchain-registry/superchain"
 )
 
+var OPStackSupport = params.ToProtocolVersion([8]byte{}, 3, 1, 0, 1)
```

Comment:
Can you add a godoc to make this easier for humans to read or can we have a more human readable interface for creating this global? Will make updating it much easier

Reply:
added a V0 struct definition, so inputs can be declared by name, and then encoded with a `.Encode()` call